configure.in

```diff
@@ -7263,6 +7260,10 @@ if test "$USE_WIN32" = 1 ; then
         fi
     fi
 
+    if test "$wxUSE_DIB" = "yes"; then
+        AC_DEFINE(wxUSE_WXDIB)
```

Comment:
I will work on trying to get the minimal sample to run without issue.
With wxUSE_WXDIB=0 and with wxHAS_IMAGES_IN_RESOURCES undefined.

@kosenko Please tell me if this is NOT the correct choice.

Tim S.